site/src/components/Dashboard/DeploymentBanner/DeploymentBannerView.tsx

```diff
@@ -124,24 +170,51 @@ export const DeploymentBannerView: FC<DeploymentBannerViewProps> = ({
         whiteSpace: "nowrap",
       }}
     >
-      <Tooltip title="Status of your Coder deployment. Only visible for admins!">
-        <div
-          css={css`
-            display: flex;
-            align-items: center;
-
-            & svg {
-              width: 16px;
-              height: 16px;
-            }
-
-            ${theme.breakpoints.down("lg")} {
-              display: none;
-            }
-          `}
-        >
-          <RocketIcon />
-        </div>
+      <Tooltip
+        classes={{ tooltip: statusSummaryStyle }}
+        title={
+          unhealthy ? (
+            <>
+              <HelpTooltipTitle>
+                We have detected problems with your Coder deployment.
+              </HelpTooltipTitle>
+              <Stack spacing={1}>
+                {health.access_url && (
+                  <HealthIssue>
+                    Your access URL may be configured incorrectly.
+                  </HealthIssue>
+                )}
+                {health.database && (
+                  <HealthIssue>Your database is unhealthy.</HealthIssue>
+                )}
+                {health.derp && (
+                  <HealthIssue>
+                    Your DERP is like, gonked up or something.
```

Comment:
This description is taco-worthy, but I feel like, for a UI where some users are likely not to speak English as a first language, it'd help to put some other phrasing in parentheses to give them more descriptive information

Reply:
And as fun as it is, I feel like it might go against Kyle's recent pushes to "genericize" Coder so that it works with any customer's brand identity

Reply:
It was just a placeholder 😅